raiden/transfer/mediated_transfer/events.py

```diff
@@ -176,3 +184,30 @@ class EventRouteFailed(Event):
     secrethash: SecretHash
     route: List[Address]
     token_network_address: TokenNetworkAddress
+
+
+@dataclass
+class EventWrapper:
+    """
+    Some events have missing data and need to be encapsulated inside wrapping events that will
```

Comment:
data to metadata (or even address_metadata) to not get confused

Reply:
This was supposed to be a generic class for wrappers. For now it's treating only metadata, but it can be used for any case that's blocking.

Reply:
Okay. If it's more general then I think the right right location for this class should be more top  level rather then `
raiden/transfer/mediated_transfer/events.py` . ( I think it should be anyway as it does not only treat mediated transfer events)

Reply:
You're right, I moved it up ✅